### PR TITLE
feat: integrate operator into workspace

### PR DESCRIPTION
**Pull request by nathanielc**
Prior to this change, the operator and runner projects were separate cargo projects. Now they are a singular workspace and the yaml is updated to deploy the operator into k8s.

### CODE REVIEW

File: operator/src/network.rs [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    let client: Client = context.client.clone();
+    let net_api: Api<Network> = Api::all(client);
+    let name = network.name_any();
+
+    let status = json!({
+        "status": NetworkStatus { enabled: true }
+    });
+
+    // TODO fails, assuming status/resource was not created
+    net_api
```
Comment (zachferland) [inline operator/src/network.rs:67]: noticed this was properly formatted/linted now, what should i be using locally during development?

File: k8s/simulation/network.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
-apiVersion: "simulator.io/v1"
+apiVersion: "keramik.3box.io/v1"
 kind: Network
 metadata:
-  name: network
-  namespace: ceramic
+  name: bob
```
Comment (zachferland) [inline k8s/simulation/network.yaml:4]: is the name/namespace set elsewhere?

**Review by zachferland — APPROVED** [review]
looks good 👍

File: operator/src/network.rs (hunk 1 above)
Comment (nathanielc) [reply operator/src/network.rs:67]: You can configure vscode to format using the rust analyzer on save. That is what I typically do (except with vim)

Alternatively you can run `cargo fmt` to format all files in the workspace.

File: k8s/simulation/network.yaml (hunk 2 above)
Comment (nathanielc) [reply k8s/simulation/network.yaml:4]: Yes, via kustimozation.yaml, this way the namespace is configurable for all related resources in one place.

**Pull request merged**